tasks/main.yml

```diff
@@ -197,3 +197,19 @@
     name: "{{ golem_source_unit_name }}"
     state: started
     enabled: yes
+
+- name: configure resources
+  when: golem_source_resource_settings
+  block:
+    - name: waiting for golem to become ready
+      shell: "{{ golem_source_cli }} account info | grep Golem_ID | awk '{print $2}'"
```

Comment:
There's no golem_source_cli variable in defaults.yml

Reply:
it's in vars

Reply:
it's added by #12